styles.css

```diff
@@ -338,8 +498,765 @@ footer {
   height: 7.55vw;
 }
 
-#image-projects img {
+#main-background1 img {
   width: 100vw;
   object-fit: cover;
-  padding-bottom: 10vw;
+  padding-bottom: 5vw;
+}
+
+#main-background2 {
+  display: none;
+}
+
+#dot-grid-template {
+  display: none;
+}
+
+.footer-resume-desktop {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .nav-bar {
+    display: flex;
+    background-color: white;
+  }
+
+  header {
+    display: none;
+  }
+
+  .title-bar {
+    display: flex;
+    flex-direction: row;
+    align-content: space-between;
+    align-items: center;
+    gap: 2.5vw;
+    padding-left: 37vw;
+    list-style: none;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.6vw;
+    line-height: 24px;
+    text-align: right;
+    color: #1a2236;
+  }
+
+  .title-bar a {
+    text-decoration: none;
+    color: #1a2236;
+  }
+
+  .headline {
+    width: 110vw;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .main {
+    display: flex;
+    flex-wrap: wrap;
+    width: 41vw;
+    height: 30vw;
+    margin-top: 12vw;
+    margin-left: 12vw;
+  }
+
+  .title,
+  .myname-intro,
+  .button-box,
+  .dot-grid {
+    width: 100%;
+  }
+
+  .title {
+    font-family: 'Playfair Display', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10vw;
+    line-height: 100%;
+    color: #1a2236;
+    margin-inline: 5vw;
+    padding-bottom: 2vw;
+    height: 10vw;
+  }
+
+  .myname-intro {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 0.9vw;
+    line-height: 1.5vw;
+    letter-spacing: -0.01em;
+    color: #1a2236;
+    margin-inline: 5vw;
+    margin-top: 13vw;
+  }
+
+  .button-box {
+    width: 15vw;
+    height: 4vw;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1vw;
+    line-height: 0.54vw;
+    text-align: center;
+    color: #fff;
+    cursor: pointer;
+    background: #1a2236;
+    margin-block: 2vw;
+    margin-inline: 5vw;
+  }
+
+  .dot-grid {
+    display: block;
+    margin-top: 0.4vw;
+    text-align: right;
+  }
+
+  .image-projects {
+    width: 50vw;
+  }
+
+  #main-background2 {
+    display: block;
+    width: 100%;
+  }
```

Comment:
- [ ] As per the previous reviewer's request, the background image of your headline should cover half of the viewport width. However, it appears that the image is too large and extends beyond the HTML element, resulting in horizontal scrolling on the X-axis. Kindly reduce the width and make sure horizontal scrolling is disabled.
![image](https://user-images.githubusercontent.com/91486791/230727863-ab6ef9a7-55fb-4862-afde-7690a3231847.png)
![image](https://user-images.githubusercontent.com/91486791/230728033-38ecda9c-fa73-4322-ae18-77176dddb406.png)